src/test/java/net/snowflake/client/jdbc/ResultSetLatestIT.java

```diff
@@ -120,6 +120,34 @@ public void testChunkDownloaderNoHang() throws SQLException {
     connection.close();
   }
 
+  /** This tests that the SnowflakeChunkDownloader doesn't hang when memory limits are low. */
+  @Test
+  public void testChunkDownloaderSetRetry() throws SQLException {
+    int stmtCount = 3;
+    int rowCount = 170000;
+    Connection connection = getConnection();
+    connection
+        .unwrap(SnowflakeConnectionV1.class)
+        .getSFBaseSession()
+        .setMemoryLimitForTesting(2000000);
```

Comment:
Please write large numbers in multiples of 512 or 1024. It is hard to reason that code otherwise.

Reply:
Also put the unit in comments. Ideally the set method should have the unit  as well